docs/UserGuide.md

```diff
@@ -120,7 +120,7 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [t/TAG]… [c/COMMENT]…`
 
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
+*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email telegramHandle of the 1st person to be `91234567` and `johndoe@example.com` respectively.
```

Comment:
This seems like a mistake - email telegramHandle seems ill-defined.

Reply:
changed to email address.